.storybook/preview-head.html

```diff
@@ -31,4 +31,8 @@
   .sbdocs.sbdocs-a, .sbdocs.sbdocs-p {
     font-size: 14px;
   }
+
```

Comment:
There is a bug where an empty doc block is displayed when using the component parameters to hide the code preview, so this just prevents it from displaying for now

Reply:
Maybe we should have a comment here referencing the bug? Is there an open issue for it?

Reply:
Yeah I did think about that after raising the PR. I can't see an open issue for it, need t get round t raising one for it also